app/components/ui/atoms/Textarea.md

```diff
@@ -0,0 +1,15 @@
+Required props:
+
+* `label`: for accessibility purposes
+* `value` & `onChange`: to make this a controlled React component
+
+Note: other CSS properties for an input element may be passed in (e.g. placeholder) but are not required.
+
+```js
+<Textarea
```

Comment:
It seems you can use `state` and `initialState` in react styleguidist markdown files[1]. Now that you removed the default value for `value` and `initialState` from this file, will this work?

https://react-styleguidist.js.org/docs/documenting.html#writing-code-examples

Reply:
See the lines below this one
`onChange` sets the state
`value` passes in the state

It works. Have you tried it?

Reply:
No, that's why I asked you :) I think asking people to checkout to a new branch to test if a PR works is too much work. Everyone should test their own changes to make sure they work before doing a PR (even though that doesn't happen all the time in this repo for some reason!). I was just making sure you tested. If it works, it's all good :)